lib/apartment/railtie.rb

```diff
@@ -39,11 +39,13 @@ def connection
         super.tap do
           Apartment::Tenant.init_once
```

Comment:
I think ```super``` shouldn't be guarded by ActiveRecord::NoDatabaseError.

it breaks my case where I expect the connection will raise ActiveRecord::NoDatabaseError, but now it returns nil. The same for ```arel_table```